tests/unit/test_io_hdf5_h5tools.py

```diff
@@ -1577,7 +1577,7 @@ def test_link(self):
 
             self.assertTrue(self.foo2.my_data.valid)  # test valid
             self.assertEqual(len(self.foo2.my_data), 5)  # test len
-            self.assertEqual(self.foo2.my_data.shape, (5,))  # test getattr with shape
+            self.assertEqual(self.foo2.my_data.size, (5,))  # test getattr with size
```

Comment:
```suggestion
            self.assertEqual(self.foo2.my_data.shape, (5,))  # test getattr with shape
```
Can we revert this change?